tests/regression/anomaly/test_anomaly_classificaiton.py

```diff
@@ -0,0 +1,214 @@
+"""Tests for Anomaly Classification with OTX CLI."""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import json
+import os
+import random
+from pathlib import Path
+from timeit import default_timer as timer
+
+import pytest
+
+from otx.cli.registry import Registry
+from tests.regression.regression_test_helpers import (
+    ANOMALY_DATASET_CATEGORIES,
+    TIME_LOG,
+    get_result_dict,
+    load_regression_configuration,
+)
+from tests.test_suite.e2e_test_system import e2e_pytest_component
+from tests.test_suite.run_test_command import (
+    nncf_eval_testing,
+    nncf_optimize_testing,
+    otx_deploy_openvino_testing,
+    otx_eval_compare,
+    otx_eval_deployment_testing,
+    otx_eval_openvino_testing,
+    otx_export_testing,
+    otx_train_testing,
+    pot_eval_testing,
+    pot_optimize_testing,
+)
+
+# Configurations for regression test.
+TASK_TYPE = "anomaly_classification"
+SAMPLED_ANOMALY_DATASET_CATEGORIES = random.sample(ANOMALY_DATASET_CATEGORIES, 3)
+
+otx_dir = os.getcwd()
+templates = Registry("otx/algorithms/anomaly").filter(task_type=TASK_TYPE.upper()).templates
+templates_ids = [template.model_template_id for template in templates]
+
+result_dict = get_result_dict(TASK_TYPE)
+result_dir = f"/tmp/regression_test_results/{TASK_TYPE}"
+Path(result_dir).mkdir(parents=True, exist_ok=True)
+
+# Detection
```

Comment:
I think these kind of comments contain wrong information and useless, how about removing theses comments

Reply:
100% Agree, Thanks